docs/_static/network/v1.yaml

```diff
@@ -277,6 +292,26 @@ components:
         softwareVersion:
           description: Indication of the software version of the node. It's recommended to use a (Git) commit ID that uniquely resolves to a code revision, alternatively a semantic version could be used (e.g. 1.2.5).
           type: string
+    Contact:
+      type: object
+      description: Describes the contact information of a node.
+      required:
+        - address
+        - attempts
+      properties:
+        address:
+          description: Address of the node.
+          type: string
+        did:
+          description: DID of the node.
+          type: string
+        attempts:
+          description: Number of times the node has been contacted, but the connection didn't succeed.
```

Comment:
Currently this is the number of connection attempts since the node has (re-)started.